src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,189 @@
+package duke;
+
+import duke.exception.*;
```

Comment:
The asterisk in this import implies it is not explicit. Should it be changed to an explicit dukeException import?